examples/webgl2_texture2darray_compressed.html

```diff
@@ -119,8 +80,41 @@
 							depth: { value: 55 },
 							size: { value: new THREE.Vector2( planeWidth, planeHeight ) }
 						},
-						vertexShader: document.getElementById( 'vs' ).textContent.trim(),
-						fragmentShader: document.getElementById( 'fs' ).textContent.trim(),
+						vertexShader: /* glsl */`
```

Comment:
TBH, I don't think it's necessary to move the GLSL code. We define it in script tags in other examples, too.

Reply:
Done, I reverted the `x-shader` code.